src/Sentry/Sentry.csproj

```diff
@@ -55,4 +55,9 @@
     <PackageReference Include="System.Text.Json" Version="5.0.2" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Content Include="buildTransitive\Sentry.props" PackagePath="buildTransitive\Sentry.props" />
+    <Content Include="buildTransitive\Sentry.props" PackagePath="build\Sentry.props" />
```

Comment:
I'm seeing two warnings in the build output related to these:

```
buildTransitive/Sentry.props : warning XA0101: @(Content) build action is not supported [/Users/matt/Code/getsentry/sentry-dotnet/src/Sentry/Sentry.csproj]
buildTransitive/Sentry.props : warning XA0101: @(Content) build action is not supported [/Users/matt/Code/getsentry/sentry-dotnet/src/Sentry/Sentry.csproj]
```

Reply:
fixed